test/Interop/Cxx/operators/member-inline-silgen.swift

```diff
@@ -32,3 +32,54 @@ public func call(_ wrapper: inout AddressOnlyIntWrapper) -> Int32 { wrapper() }
 // CHECK: end_access [[SELFACCESS]] : $*AddressOnlyIntWrapper
 
 // CHECK: sil [clang AddressOnlyIntWrapper.callAsFunction] [[NAME]] : $@convention(c) (@inout AddressOnlyIntWrapper) -> Int32
+
+public func index(_ arr: inout ReadOnlyIntArray, _ arg: Int32) -> Int32 { arr[arg] }
+
+// CHECK: sil @[[FUNCNAME:.*]] : $@convention(thin) (@inout ReadOnlyIntArray, Int32) -> Int32 {
+// CHECK: bb0([[SELF:%.*]] : $*ReadOnlyIntArray, [[INDEX:%.*]] : $Int32):
```

Comment:
This is checking `index(arr:)`, right? So, this wouldn't be `SELF` it would be `ARR` or `ARG0` or something.

Reply:
You're right, fixed it.